ci-operator/jobs/openshift/cluster-node-tuning-operator/openshift-cluster-node-tuning-operator-master-presubmits.yaml

```diff
@@ -1,5 +1,62 @@
 presubmits:
   openshift/cluster-node-tuning-operator:
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - master
+    context: ci/prow/e2e-aws
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    name: pull-ci-openshift-cluster-node-tuning-operator-master-e2e-aws
+    rerun_command: /test e2e-aws
+    spec:
+      containers:
+      - args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --give-pr-author-access-to-namespace=true
+        - --secret-dir=/usr/local/e2e-aws-cluster-profile
+        - --target=e2e-aws
+        - --template=/usr/local/e2e-aws
+        command:
+        - ci-operator
+        env:
+        - name: CLUSTER_TYPE
+          value: aws
+        - name: CONFIG_SPEC
+          valueFrom:
+            configMapKeyRef:
+              key: openshift-cluster-node-tuning-operator-master.yaml
+              name: ci-operator-configs
+        - name: JOB_NAME_SAFE
+          value: e2e-aws
+        - name: TEST_COMMAND
+          value: TEST_SUITE=openshift/conformance/parallel run-tests
```

Comment:
I see some other operators just calling `make test-e2e`, is `TEST_SUITE=openshift/conformance/parallel run-tests` intended here?

Reply:
This depends on the template - this is based on `cluster-launch-installer-e2e` template, which runs the "shared" openshift e2e tests from `openshift/origin` repo. The ones that have something like `make test-e2e` here are usually based on `cluster-launch-installer-src` template (note the `-src` suffix compared to `-e2e` suffix above), which runs tests defined in the **target** repository, usually by calling some make target.

To make this even more fun, Prow job config files (the ones under `ci-operator/jobs`) are usually generated from the ci-operator config files (the ones under `ci-operator/config`), so this whole hunk was actually generated from the little hunk in the other file and therefore it's very likely to be correct ;) The decision which template to use is driven by https://github.com/openshift/release/pull/2829/files#diff-80de3886ec4357e9b24ff58898a3b191R39 and the command is given on https://github.com/openshift/release/pull/2829/files#diff-80de3886ec4357e9b24ff58898a3b191R38, everyhting else is generated :)

Reply:
Many thanks!  So this basically means the some operators are basically skipping the shared e2e tests.  For this operator, this means additional, "shared" checks, which is good, but also bad as the chance for hitting issues not caused by the operator changes increases, if I understand this correctly.